### PR TITLE
Add Auction Tour

### PR DESCRIPTION
**Pull request by Namaskar-1F64F**
image
image
image
image
image
image

### CODE REVIEW

File: src/components/auction/AuctionBody/index.tsx [hunk 1]
```diff
@@ -136,6 +143,9 @@ const BondCard = ({ graphInfo }: { graphInfo: Auction }) => {
 
 const AuctionBody = (props: AuctionBodyProps) => {
   const { auctionIdentifier, derivedAuctionInfo, graphInfo } = props
+  const auctionInformationRef = createRef<HTMLDivElement>()
+  const bondInformationRef = createRef<HTMLDivElement>()
```
Comment (RusseII) [inline src/components/auction/AuctionBody/index.tsx:147]: wow haven't seen this very often!

File: src/components/auction/AuctionTour/index.tsx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+              </a>
+              .
+            </span>
+          </p>
+          <p className="pb-8">
+            <span>
+              As always, we have documentation on our{' '}
+              <a
+                className="text-[#6CADFB]"
+                href="https://docs.arbor.finance"
```
Comment (RusseII) [inline src/components/auction/AuctionTour/index.tsx:53]: link to auction page probably

**Review by RusseII — APPROVED** [review]

**Review by RusseII — CHANGES_REQUESTED** [review]
does not work on smaller screen sizes :(

File: src/components/auction/AuctionTour/index.tsx (hunk 2 above)
Comment (Namaskar-1F64F) [reply src/components/auction/AuctionTour/index.tsx:53]: the walkthrough link is right under it which links to the auction page

**Pull request merged**